PostHog/Replay/URLSessionSwizzler.swift

```diff
@@ -60,192 +60,192 @@
 
         /// Swizzles the `URLSession.dataTask(with:completionHandler:)` for `URLRequest`.
         class DataTaskWithURLRequestAndCompletion: MethodSwizzler<
-        @convention(c) (URLSession, Selector, URLRequest, CompletionHandler?) -> URLSessionDataTask,
+            @convention(c) (URLSession, Selector, URLRequest, CompletionHandler?) -> URLSessionDataTask,
```

Comment:
If there was no changes here, you can rollback, the linter did it?

Reply:
> If there was no changes here, you can rollback, the linter did it?

all the changes to this file was done by `make swiftFormat`

Reply:
this is strange because the submitted one fails the ci.

i guess it might be due to versioning. let me rollback.

Reply:
yeah could be because the CI is happy with the current version.